doc/usage.md

```diff
@@ -168,7 +168,7 @@ Every `Error` object we pass back has the properties:
 For example, sending message to a non-existent user:
 
 ```js
-bot.sendMessage(nonExistentUserId, 'text').catch(error => {
```

Comment:
I have just tried it with 6.9.5 to confirm, and it does work. The syntax is supported.

You reported that your IDE is logging the errors. You sure it is using an updated Node.js?